src/mysky/index.ts

```diff
@@ -0,0 +1,160 @@
+export type { CustomConnectorOptions } from "./connector";
+export { DacLibrary } from "./dac";
+
+import { PermCategory, PermHidden, Permission, PermType, PermWrite } from "skynet-interface-utils";
+
+import { Connector, CustomConnectorOptions } from "./connector";
+import { SkynetClient } from "../client";
+import { DacLibrary } from "./dac";
+import { RegistryEntry, setSignedEntry, SignedRegistryEntry } from "../registry";
+import { CustomGetJSONOptions, CustomSetJSONOptions, getOrCreateRegistryEntry, JsonData } from "../skydb";
+import { hexToUint8Array } from "../utils/string";
+import { Signature } from "../crypto";
+import { deriveDiscoverableTweak } from "./tweak";
+
+export async function loadMySky(
+  this: SkynetClient,
+  appPath: string,
+  customOptions?: CustomConnectorOptions
+): Promise<MySky> {
+  const mySky = await MySky.init(this, customOptions);
+  mySky.addPermissions(new Permission(appPath, PermHidden, PermWrite));
+
+  return mySky;
+}
+
+export const mySkyDomain = "skynet-mysky.hns";
```

Comment:
(nit) constants are usually defined using all caps, or prefixed with `CONST_`, not doing so is a little annoying when reading the code because you don't know where the variable comes from. At first sight this is just a variable and you won't be able to find it in the function scope, or w/e it's used.

Reply:
You are right, but we already export constants using this naming scheme 🤷‍♂️ We can change that with a compatibility break since we're doing a new major version anyway, but not sure if it's worth our time right now.

Reply:
I agree it's low priority, that's also why I tagged it as `(nit)`, maybe we can resolve in an issue?


I did a quick search in `skynet-js` for `export const`
```
src/download.ts:
  79: export const defaultDownloadOptions = {
  80    ...defaultOptions("/"),

src/registry.ts:
  26: export const defaultGetEntryOptions = {
  27    ...defaultOptions("/skynet/registry"),
  35: export const MAX_GET_ENTRY_TIMEOUT = 300; // 5 minutes
  40: export const regexRevisionNoQuotes = /"revision":\s*([0-9]+)/;

src/mysky/index.ts:
  26: export const mySkyDomain = "skynet-mysky.hns";
  27  const mySkyUiRelativeUrl = "ui.html";

src/utils/number.ts:
  4: export const MAX_REVISION = BigInt("18446744073709551615"); // max uint64

src/utils/skylink.ts:
  33: export const uriHandshakePrefix = "hns:";
  34: export const uriHandshakeResolverPrefix = "hnsres:";
  35: export const uriSkynetPrefix = "sia:";

src/utils/url.ts:
  10: export const defaultSkynetPortalUrl = "https://siasky.net";
  ```
 
doesn't really like we're consistently using a schema at the moment anyway. Changing it doesn't even have to break compatibility either, we could just do something like
  
  ```
  // @deprecated please use URI_HANDSHAKE_PREFIX
  export const uriHandshakePrefix = "hns:"

  export const URI_HANDSHAKE_PREFIX = "hns:"
  ```
  and then just adopt `URI_HANDSHAKE_PREFIX` everywhere. Leaving this here just as context for the issue, can resolve.

Reply:
https://github.com/SkynetHQ/skynet-js/issues/24